debian/control

```diff
@@ -21,4 +21,4 @@ X-Python3-Version: >= 3.6
 Package: clickhouse-tools
 Architecture: any
 Description: A set of tools for administration and diagnostics of ClickHouse DBMS.
-Replaces: mdb-ch-tools
```

Comment:
It's better to keep`Replaces: mdb-ch-tools`.

But I don't known what exact syntax should be used.

```
Replaces: mdb-ch-tools,ch-tools
```

or

```
Replaces: mdb-ch-tools
Replaces: ch-tools
```

Reply:
Yes it is allowable `Replaces: package-list`